src/trans/gpu/algor/ext_acc.F90

```diff
@@ -323,12 +321,11 @@ subroutine ext_acc_copyout(ptrs, stream)
 
     do i = 1, num_ranges
       call c_f_pointer(common_ptrs(i)%ptr, pp, shape=[common_ptrs(i)%sz/c_sizeof(pp(1))])
-      !!call acc_copyout_async(pp, common_ptrs(i)%sz, async=stream_act)
-      call acc_copyout(pp, int(common_ptrs(i)%sz))
+      !$acc exit data copyout(pp) async(stream_act)
     enddo
   end subroutine
   subroutine ext_acc_delete(ptrs, stream)
-    use openacc, only: acc_async_sync, acc_delete
```

Comment:
Why remove `acc_delete`? It is still called below.

Reply:
that is a stupid typo... it should not be called. I had to switch to clauses because acc_delete only takes int32, not int64. This code deallocated nothing :D  I will test if everything properly deallocates after #151 has been merged).

Reply:
this is fine now. acc_delete is not called - we are using pragmas because they allow for size_t.